src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -1127,6 +1127,15 @@ public String execute(Jedis connection) {
       }
     }.run(null);
   }
+  @Override
+  public String readonly() {
```

Comment:
What's the purpose of adding the readonly command here?. JedisCluster will throw an error because there's no key supplied in the `run` method.